test/CMakeLists.txt

```diff
@@ -106,6 +106,10 @@ add_fmt_test(printf-test)
 add_fmt_test(ranges-test)
 add_fmt_test(scan-test)
 
+
+add_fmt_test(enforce-compiletime-test)
```

Comment:
enforce-compiletime-test -> enforce-compile-string-test for consistency with the macro name